cloudformation/netagent2/banyan-access-tier-v2-amazon-linux.json

```diff
@@ -0,0 +1,439 @@
+{
+  "AWSTemplateFormatVersion": "2010-09-09",
+  "Description": "Banyan Elastic Access Tier v2",
+  "Metadata": {
+    "AWS::CloudFormation::Interface": {
+      "ParameterGroups" : [
+        {
+          "Label": { "default": "Banyan Configuration" },
+          "Parameters": ["ApiKeySecret", "CommandCenterURL", "BanyanPackage"]
```

Comment:
Add `HTTPSProxy` back?

Reply:
I actually looked at the V1 templates and it looks like it's not available there but I think it might be a good idea to maintain parity between all our install methods anyway.

Reply:
Maybe in a subsequent PR.